octobot_trading/enums.py

```diff
@@ -366,6 +366,16 @@ class ExchangeConstantsLeveragePropertyColumns(enum.Enum):
     RAW = "raw"
 
 
+class ExchangeConstantsLeverageTiersColumns(enum.Enum):
+    TIER = "tier"
+    CURRENCY = "currency"
+    MIN_NOTIONAL = "minNotional"
```

Comment:
can we use snake_case for octobot unified enums plz ? try to follow this principle

Reply:
done

Reply:
thanks !